compiler/damlc/lib/DA/Cli/Options.hs

```diff
@@ -338,10 +338,15 @@ enableScenarioOpt = EnableScenarioService <$>
     flagYesNoAuto "scenarios" True "Enable/disable support for running scenarios" idm
 
 hlintEnabledOpt :: Parser HlintUsage
-hlintEnabledOpt = HlintEnabled <$> strOption
+hlintEnabledOpt = HlintEnabled
+  <$> strOption
   ( long "with-hlint"
```

Comment:
We should rename this to `--with-lint` or `--with-dlint`.

Reply:
✔️